python/hsfs/core/feature_monitoring_config_engine.py

```diff
@@ -152,15 +242,33 @@ def build_feature_monitoring_config(
         statistics_comparison_config: Dict[str, Any],
         scheduler_config: str,
         alert_config: str,
-        feature_group_id: Optional[int] = None,
-        feature_view_id: Optional[int] = None,
         description: Optional[str] = None,
     ) -> FeatureMonitoringConfig:
+        """Builds a feature monitoring config.
+
+        Args:
+            feature_name: str, required
+                Name of the feature to monitor.
+            name: str, required
+                Name of the monitoring configuration.
+            detection_window_config: Dict[str, Any], required
+                Configuration of the detection window.
+            reference_window_config: Dict[str, Any], required
+                Configuration of the reference window.
+            statistics_comparison_config: Dict[str, Any], required
+                Configuration of the statistics comparison.
+            scheduler_config: str, required
+                Configuration of the scheduler.
+            alert_config: str, required
```

Comment:
alerts should be optional
```suggestion
            alert_config: str, optional
```